plugins/config/parameters.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+import (
+	flag "github.com/spf13/pflag"
+)
+
+const (
+	DisablePlugins = "node.disablePlugins"
```

Comment:
In most (if not all) modules we use `Cfg.....` naming for config consts
```suggestion
	CfgDisablePlugins = "node.disablePlugins"
```